doc/rados/operations/placement-groups.rst

```diff
@@ -459,10 +459,10 @@ let's imagine a scenario that results in permanent data loss in a single PG:
    OSD happened to contain the only remaining copy of an object, the object is
    permanently lost.
 
-In a cluster containing 10 OSDs with 512 PGs in a three- replica pool, CRUSH
-will give each PG three OSDs.  Ultimately, each OSD hosts (512 * 3) / 10 = ~150
-PGs.  So when the first OSD fails in the above scenario, recovery will begin
-for all 150 PGs at the same time.
+In a cluster containing 10 OSDs with 512 PGs in a three-replica pool, CRUSH
+will give each PG three OSDs.  Ultimately, each OSD hosts :math:`\frac{(512 *
+3)}{10} = ~150` PGs. So when the first OSD fails in the above scenario,
```

Comment:
This expression has been processed by the :math: extension and is presented in an easier-to-read, more mathematically calligraphic form. However, it is not findable by CTRL+F. Some of the mathematical expressions in this section have been processed by the :math: extension. Others have been left unaltered, so that the reader has ready and local points of comparison.

Reply:
Fair enough, though I suspect arithmetic expressions are not often the direct target of searches.